agent/acs/handler/acs_handler.go

```diff
@@ -438,48 +388,24 @@ func (acsSession *session) heartbeatJitter() time.Duration {
 	return acsSession._heartbeatJitter
 }
 
-// createACSClient creates the ACS Client using the specified URL
-func (acsResources *acsSessionResources) createACSClient(url string, cfg *config.Config) wsclient.ClientServer {
-	return acsclient.New(url, cfg, acsResources.credentialsProvider, wsRWTimeout)
-}
-
-// connectedToACS records a successful connection to ACS
-// It sets sendCredentials to false on such an event
-func (acsResources *acsSessionResources) connectedToACS() {
-	acsResources.sendCredentials = false
-}
-
-// getSendCredentialsURLParameter gets the value to be set for the
-// 'sendCredentials' URL parameter
-func (acsResources *acsSessionResources) getSendCredentialsURLParameter() string {
-	return strconv.FormatBool(acsResources.sendCredentials)
-}
-
-func newSessionResources(credentialsProvider *credentials.Credentials) sessionResources {
-	return &acsSessionResources{
-		credentialsProvider: credentialsProvider,
-		sendCredentials:     true,
-	}
-}
-
 // acsWsURL returns the websocket url for ACS given the endpoint
-func acsWsURL(endpoint, cluster, containerInstanceArn string, taskEngine engine.TaskEngine, acsSessionState sessionState) string {
+func (acsSession *session) acsWsURL(endpoint string) string {
```

Comment:
nit: "Ws" -> should be "WS" (uppercase acronyms) but I would choose shortening it to `acsURL` over `acsWSURL`